x-pack/plugins/cases/server/plugin.ts

```diff
@@ -72,7 +72,7 @@ export class CasePlugin {
     this.clientFactory = new CasesClientFactory(this.log);
   }
 
-  public async setup(core: CoreSetup, plugins: PluginsSetup) {
+  public setup(core: CoreSetup, plugins: PluginsSetup) {
```

Comment:
async plugin lifecycle functions are going to be deprecated by core and we don't do `awaits` anyway so removing.